cfgov/cfgov/settings/local.py

```diff
@@ -99,3 +99,11 @@
 # unless the GOVDELIVERY_BASE_URL environment variable is set.
 if not os.environ.get("GOVDELIVERY_BASE_URL"):
     GOVDELIVERY_API = "core.govdelivery.LoggingMockGovDelivery"
+
+# Use a placeholder image service to replace images that are uploaded to S3
+WAGTAIL_PLACEHOLDERIMAGES_DUMMY = True
+WAGTAIL_PLACEHOLDERIMAGES_SOURCE = "//placedog.net/{width}/{height}/"
```

Comment:
```suggestion
WAGTAIL_PLACEHOLDERIMAGES_SOURCE = "//placekitten.com/{width}/{height}/"
```

😛

Reply:
I'm fine with that 😸  I just left it at the default, but yeah, we do have a history of using placekitten, too, so why not?

Reply:
Var in `.env` to specify your preference of place animals 😛